README.md

```diff
@@ -1,75 +1,106 @@
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Catalog of my things
+
+> This is the final group project for Ruby to develop a console application implementing the Catalog of things displaying the list of books, musics, movies, and games. It is developed using Ruby.
+
+### Console App Screenshot
+
+![](./app_screenshot.JPG)
+
+## Built With
+
+- Ruby
+- IRB 
+- VS Code 
+
+## Live Demo 
+
+No live demo 
+
+## Video presentation 
+
+Click [here](https://drive.google.com/file/d/1xAS7NzLmWf4GhX3meFVXAUX_q0kVYJIr/view?usp=share_link) to see video presentation
+
+
+## Getting Started
+
+Copy this link https://github.com/fullstop125/catalog-of-my-things.git.
+Get the directory that you want to clone the repository.
+Open the command prompt in this directory.
+
+### Setup
+Clone repository
+
+```
+git clone https://github.com/fullstop125/catalog-of-my-things.git
+```
+
+Go to the repository folder in your command prompt
+```
+cd catalog-of-my-things
+```
+
+## Install
+Install Ruby using WSL 
+
+Install Ubuntu after system restart  
+Run the commands
+
+```
+bundle install
+```
+
+## To run project
+
+on the command prompt 
+
+```
+ruby main.rb
+```
+
+## To run tests for the application 
+
+on the command prompt 
+
+```
+rspec
+```
+
+## Authors
+
+👤 **Ahmad Zaki Yousufi**
+
+- GitHub: [@zakiyosufi](https://github.com/zakiyousufi)
+- LinkedIn: [zakiyousufi](https://linkedin.com/in/zakiyousufi)
+
+👤 **Momanyi**
+
+- GitHub: [@fullstop125](https://github.com/fullstop125)
+- Twitter: [@MosesHassany](https://twitter.com/MosesHassany)
+- LinkedIn: [momanyi-hassan](https://www.linkedin.com/in/momanyi-hassan-32a489180/)
+
+👤 **Juan Diaz**
+
+- GitHub: [@juanmanuelbyc](https://github.com/juanmanuelbyc)
+- LinkedIn: [juandiaz](www.linkedin.com/in/juandiaz1991)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.

```

Comment:
Great job team 🤝

- [ ] [Optional] You may kindly fix your MIT license link to make your Readme file more professional.